compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -4093,6 +4093,46 @@
                     "type": "Token"
                 }
             ]
+        },
+        {
+            "name": "OrderByClauseNode",
+            "base": "ClauseNode",
+            "kind": "ORDER_BY_CLAUSE",
+            "attributes": [
+                {
+                    "name": "orderKeyword",
+                    "type": "Token"
+                },
+                {
+                    "name": "byKeyword",
+                    "type": "Token"
+                },
+                {
+                    "name": "orderKey",
+                    "type": "OrderKeyNode",
+                    "occurrences": "MULTIPLE_SEPARATED"
+                }
+            ]
+        },
+        {
+            "name": "OrderKeyNode",
+            "base": "Node",
+            "kind": "ORDER_KEY",
+            "attributes": [
+                {
+                    "name": "expression",
+                    "type": "ExpressionNode"
+                },
+                {
+                    "name": "ascendingKeyword",
+                    "type": "Token"
+                },
+                {
+                    "name": "descendingKeyword",
+                    "type": "Token",
+                    "isOptional": true
+                }
```

Comment:
These two should be replaced by a single field.
eg:
```
"attributes": [
    {
        "name": "expression",
        "type": "ExpressionNode"
    },
    {
        "name": "order",
        "type": "Token"
        "isOptional": true
    }
]
```
This way, only one can exist at a time.